app/controllers/api/v1/application_controller.rb

```diff
@@ -50,6 +50,7 @@ def authenticate
 
       def current_api_user
         @current_api_user ||= authenticate
+        Current.user = @current_api_user
```

Comment:
Why can't we do `Current.api_user` here? And then set it as an attribute in `current.rb`.
This seems like it will override things